src/Authentication/Authentication/Helpers/ContentHelper.cs

```diff
@@ -80,6 +82,8 @@ private static bool CheckIsOctetStream(string contentType)
             return isOctetStream;
         }
 
+        private static bool IsPlainText(string contentType) => string.Equals(contentType.ToLower(CultureInfo.InvariantCulture), "text/plain");
```

Comment:
```suggestion
        private static bool IsPlainText(string contentType) => contentType.Equals("text/plain", StringComparison.OrdinalIgnoreCase);
```

Reply:
Is Invariant culture only applicable to parsing dates and numbers?